apps/ssr-tests-v9/src/utils/buildAssets.ts

```diff
@@ -0,0 +1,60 @@
+import { TsconfigPathsPlugin } from '@esbuild-plugins/tsconfig-paths';
+import { build, BuildOptions } from 'esbuild';
+import * as path from 'path';
+
+const NODE_MAJOR_VERSION = process.versions.node.split('.')[0];
+
+const commonOptions: BuildOptions = {
+  bundle: true,
+  jsx: 'transform',
+  loader: {
+    '.svg': 'dataurl',
+  },
+  // Used to resolve packages aliases in stories files.
+  plugins: [TsconfigPathsPlugin({ tsconfig: path.resolve(__dirname, 'tsconfig.base.json') })],
+};
+
+type BuildConfig = {
+  cjsEntryPoint: string;
+  cjsOutfile: string;
+  esmEntryPoint: string;
+  esmOutfile: string;
+};
+
+export async function buildAssets(config: BuildConfig): Promise<void> {
+  const { cjsEntryPoint, cjsOutfile, esmEntryPoint, esmOutfile } = config;
+
+  try {
+    // Used for SSR rendering, see renderToHTML.js
+    await build({
+      ...commonOptions,
+
+      entryPoints: [cjsEntryPoint],
+      outfile: cjsOutfile,
+
+      // External dependencies should not be bundled
+      external: ['@griffel/core', '@griffel/react', 'react', 'react-dom', 'scheduler'],
+      format: 'cjs',
+      target: `node${NODE_MAJOR_VERSION}`,
```

Comment:
As I know `node current` is not support as a target.

Reply:
Well it seems that it's by design...

- If I will specify just `node`:
  > Target "node" is missing a version number in "--target=node"
- If I will specify `nodecurrent`
  > ✘ [ERROR] Invalid version: "current" 

TBH not sure that `nodecurrent` is good that's why I didn't open a feature request.